ceno_zkvm/src/circuit_builder.rs

```diff
@@ -94,6 +94,10 @@ pub struct ConstraintSystem<E: ExtensionField> {
     pub num_fixed: usize,
     pub fixed_namespace_map: Vec<String>,
 
+    pub num_structural_fixed: usize,
+    pub structural_fixed_namespace_map: Vec<String>,
+    pub structural_fixed_len: Vec<usize>,
```

Comment:
I think we dont need to define new `structural_fixed`. Instead we can just reuse current `structural_witin`?